server.js

```diff
@@ -114,12 +114,11 @@ async function storeCard(req, res) {
 
       logger.info('Store Card succeeded!', { result, statusCode });
 
-      // remove 64-bit values from card object.
-      // card is serialized by send method
-      // which will fail to serialize if response includes 64-bit value
-      delete result.card.expMonth;
-      delete result.card.expYear;
-      delete result.card.version;
+      // cast 64-bit values to string
+      // to prevent JSON serialization error during send method
+      result.card.expMonth = result.card.expMonth.toString();
+      result.card.expYear = result.card.expYear.toString();
+      result.card.version = result.card.version.toString();
```

Comment:
How do the final strings look? (Just want to make sure the serialization looks reasonable)

Reply:
Got you!  Using info from the sandbox test cards, the relevant fields look like this once serialized
```
card: {
  ...
     expMonth: "11",
     expYear: "2025",
     version: "1"
  ...
}
```

Reply:
Alright, that seems fine-ish.